code/modules/mining/machine_redemption.dm

```diff
@@ -37,21 +38,22 @@
 
 /obj/machinery/mineral/ore_redemption/RefreshParts()
 	var/ore_pickup_rate_temp = 15
+	var/point_upgrade_temp = 1
 	var/sheet_per_ore_temp = 1
 	for(var/obj/item/stock_parts/matter_bin/B in component_parts)
-		sheet_per_ore_temp = 0.65 + (0.15 * B.rating)
-	for(var/obj/item/stock_parts/micro_laser/L in component_parts)
```

Comment:
What is this for

Reply:
As in, why are you removing it

Reply:
Mmh nevermind, you're using them for amount of points gained, I think matter bins shouldn't give you the amount laser + bins would give though.

Reply:
Yeah that was swapped out for another bonus. Specifically the same bonus as back when mining points originally existed.
Also mining points can be converted to credits, as I have readded the space credits item to the vendor. I'll actually update the PR to mention that